ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -512,6 +546,13 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
                       SystemClock.nanoTime(),
                       editText.getText().toString()));
             }
+            if (actionId == EditorInfo.IME_ACTION_NEXT ||
```

Comment:
I'm not sure we want to do special handling on this.  Or at least do it as a separate request.